pkgs/development/embedded/nmrpflash/default.nix

```diff
@@ -0,0 +1,37 @@
+{ fetchFromGitHub
+, gcc
+, git
+, lib
+, libnl
+, libpcap
+, pkg-config
+, stdenv
+}:
+stdenv.mkDerivation rec {
+  pname = "nmrpflash";
+  version = "0.9.16";
+
+  src = fetchFromGitHub {
+    owner  = "jclehner";
+    repo   = "nmrpflash";
+    rev    = "v${version}";
+    sha256 = "sha256-0nqdbXf1syUe7o5hoNIKLruKxkNaUsGolfZzoQY15j4==";
+  };
+
+  nativeBuildInputs = [ git pkg-config ];
```

Comment:
Why do we need git?

Reply:
The Makefile uses it to determine the version code.

Reply:
We should patch that. The source is missing the .git directory anyway so that will probably not work.